Source/WebKit/NetworkProcess/Downloads/cocoa/DownloadCocoa.mm

```diff
@@ -122,6 +134,55 @@
 #endif
     }
 }
+
+void Download::setPlaceholderURL(NSURL *placeholderURL)
+{
+    if (!placeholderURL)
+        return;
+
+    BOOL usingSecurityScopedURL = [placeholderURL startAccessingSecurityScopedResource];
+
+    NSError *error = nil;
+    NSData *data = [placeholderURL bookmarkDataWithOptions:(1<<24) includingResourceValuesForKeys:nil relativeToURL:nil error:&error];
```

Comment:
Can we give 1<<24 a name and maybe use an SPI header at some point?

Reply:
This option is not required, and I have removed it in the latest patch.